### PR TITLE
Remove unneeded references to execDriver

### PR DESCRIPTION
**Pull request by mlaventure**
This includes:
- updating the docs
- removing dangling variables

Signed-off-by: Kenfe-Mickael Laventure mickael.laventure@gmail.com
## 

Trying to make master docs less confusing as they still refer to execdriver atm.

hasmter

### CODE REVIEW

**Comment by MHBauer** [comment]
followup to 20662? @thaJeztah?

File: daemon/volumes_windows.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ import (
 
 // setupMounts configures the mount points for a container by appending each
 // of the configured mounts on the container to the oci mount structure
-// which will ultimately be passed into the exec driver during container creation.
+// which will ultimately be passed into the oci runtime during container creation.
```
Comment (MHBauer) [inline daemon/volumes_windows.go]: Maybe capitalize OCI.

File: docs/admin/configuring.md [hunk 2]
```diff
@@ -162,7 +162,7 @@ can be located at `/var/log/upstart/docker.log`
 
     $ tail -f /var/log/upstart/docker.log
     INFO[0000] Loading containers: done.
-    INFO[0000] docker daemon: 1.6.0 4749651; execdriver: native-0.2; graphdriver: aufs
+    INFO[0000] Docker daemon commit=1b09a95-unsupported graphdriver=aufs version=1.11.0-dev
```
Comment (MHBauer) [inline docs/admin/configuring.md]: We might want to revisit this in the near future at 1.11 release so that it looks `right` to an end user.

**Comment by MHBauer** [comment]
Also a bunch of noise in terms of the end of line spaces being removed. I'm fine with it, but it makes it more difficult to review.

File: docs/admin/configuring.md (hunk 2 above)
Comment (thaJeztah) [inline docs/admin/configuring.md]: Or just fake it :+1:

Comment (mlaventure) [inline docs/admin/configuring.md]: This is the actual output of the daemon with containerd atm :)

File: daemon/volumes_windows.go (hunk 1 above)
Comment (mlaventure) [inline daemon/volumes_windows.go]: It should have been indeed, thanks

**Comment by mlaventure** [comment]
Yes, sorry for the whitespace changes. I wish github had an option to hide those like with `git diff -w`

**Comment by thaJeztah** [comment]
Actually, it has; add `?w=1` to the url; https://github.com/docker/docker/pull/21367/files?w=1

**Comment by MHBauer** [comment]
beat me to it. I remembered after @mlaventure mentioned diff.

**Comment by calavera** [comment]
LGTM

**Comment by tiborvass** [comment]
LGTM

**Pull request merged**